infrastructure/quick-deploy/aws/metrics-server/locals.tf

```diff
@@ -11,5 +11,7 @@ locals {
     "--kubelet-use-node-status-port",
     "--metric-resolution=15s"
   ] : var.args
-  host_network = try(var.host_network, false)
+  host_network          = try(var.host_network, false)
+  helm_chart_repository = try(var.helm_chart_repository, "https://kubernetes-sigs.github.io/metrics-server/")
```

Comment:
This try cannot fail as the `var.helm_chart_repository` will always be valid (even if empty or null)